covid19_sfbayarea/data/san_francisco.py

```diff
@@ -152,12 +153,16 @@ def get_age_table(session : SocrataApi, resource_ids: Dict[str, str]) -> List[Di
     AGE_KEYS = {"18_and_under": "under 18", "18_to_30": "18-30", "31_to_40": "31-40", "41_to_50": "41-50",
                 "51_to_60": "51-60", "61_to_70": "61-70", "71_to_80": "71-80", "81_and_older": "81+",}
 
+    # find the latest date of data collection
+    params = {'$select': 'max(specimen_collection_date) as date'}
+    latest_date = session.resource(resource_id, params=params)[0]
```

Comment:
I think this all might read a bit more clearly if `latest_date` was the actual date string instead of a dict:

```suggestion
    latest_date = session.resource(resource_id, params=params)[0]['date']
```